includes/payments/class-payments-query.php

```diff
@@ -512,43 +512,19 @@ public function give_forms() {
 			return;
 		}
 
-		global $give_logs;
-
-		$args = array(
```

Comment:
I see you're cleaning up the code here, which is nice. Have you tested this throughout the ensure it works as expected?